components/gfx/platform/windows/font.rs

```diff
@@ -40,11 +42,211 @@ impl FontTableMethods for FontTable {
     }
 }
 
+fn make_tag(tag_bytes: &[u8]) -> FontTableTag {
+    assert!(tag_bytes.len() == 4);
```

Comment:
nit: If you use `assert_eq!` this will display the length in the eventuality that it fails.